docs/extensions/resources/worker.md

```diff
@@ -122,6 +124,8 @@ The provider extension (respectively, machine-controller-manager) is still respo
 
 The `spec.pools[].nodeTemplate.capacity` field contains the resource information of the machine like `cpu`, `gpu`, and `memory`. This info is used by Cluster Autoscaler to generate `nodeTemplate` during scaling the `nodeGroup` from zero.
 
+The `spec.pools[].nodeTemplate.virtualCapacity` field contains the _virtual_ resource information associated with the machine and used to specify extended resources that are _virtual_ in nature (For specifying real, provisionable resources `nodeTemplate.capacity` should be used). This will be applied to the machine class `nodeTemplate` without triggering a rollout of the cluster and will be  used by Cluster Autoscaler for scaling the `nodeGroup` from zero.
```

Comment:
```suggestion
The `spec.pools[].nodeTemplate.virtualCapacity` field contains the _virtual_ resource information associated with the machine and used to specify extended resources that are _virtual_ in nature (for specifying real, provisionable resources, `nodeTemplate.capacity` should be used). This will be applied to the machine class `nodeTemplate` without triggering a rollout of the cluster and will be  used by Cluster Autoscaler for scaling the `nodeGroup` from zero.
```